app/components/Lightbox.tsx

```diff
@@ -998,7 +1029,7 @@ const StyledImg = styled.img<{
   object-fit: contain;
   cursor: ${(props) =>
     props.$panning
-      ? "grab"
+      ? "grabbing"
```

Comment:
Oh, this resembles "closed fist". I feel `grab` was better(image wiping your monitor screen). Not too picky with this though, just a personal choice.